web-components/src/components/combobox/ComboBox.ts

```diff
@@ -1089,6 +1088,7 @@ export namespace ComboBox {
               this.handleSelectAll();
             }
           }
+          this.expanded = true;
```

Comment:
this.expanded is false then it will not go inside if block? Why we want to do this?

Reply:
currently, even if the combo box is not open space is toggling the selection of the item before opening the Combobox. This is to ensure that the selection or deselection of the list item works only once the Combobox is opened.